app/scripts/filters/resources.js

```diff
@@ -120,16 +120,29 @@ angular.module('openshiftConsole')
       });
       return nameCount;
     }
-    return function (resource, projects){
+    function escapeLessThanGreaterThan(str) {
+      return str.
+        replace(/</g, '&lt;').
+        replace(/>/g, '&gt;');
+    }
```

Comment:
I just checked and there's a problem with `&`. Set the project display name to an HTML entity to see. But Angular sanitizes things like `<script>` tags.